src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -53,8 +54,9 @@ internal WorkerProcess(IScriptEventManager eventManager, IProcessRegistry proces
 
         internal Queue<string> ProcessStdErrDataQueue => _processStdErrDataQueue;
 
-        // for testing
-        internal Process Process { get; set; }
+        public Process Process { get; set; }
```

Comment:
Why did we end up making this public?

Reply:
We are using it in another class JobObjectRegistry to check the status of process termination.